app_web/index.html

```diff
@@ -168,30 +150,28 @@ <h2>Models</h2>
                 </b-tab-item>
 
 
-                <b-tab-item label="Display" icon="library-music" class="tab-item tabItemScrollable">
+                <b-tab-item label="Display" icon="library-music" class="tabItemScrollable tab-item">
                     <template #header>
                         <!-- to get colored icons use: https://stackoverflow.com/a/43916743 -->
                         <img src="assets/ui/Display 30X30.svg" width="30px" height="30px">
                         <span>Display</span>
                     </template>
 
                     <div class="tabContent">
-                        <h2>Display</h2>
+                        <h2 class="title">Display</h2>
                         <section>
-                            <label class="label">Lighting</label>
+                            <label class="subtitle">Lighting</label>
                             <b-switch v-model="ibl" v-stream:input="iblChanged$">Image Based</b-switch>
```

Comment:
somehow this switch uses the same line as the "Lighting" label for me
<img width="272" alt="Screenshot 2021-02-03 at 11 07 54" src="https://user-images.githubusercontent.com/39583592/106731815-3f279280-6610-11eb-8a80-9309fd6a9e27.png">